config/initializers/dalli.rb

```diff
@@ -0,0 +1,23 @@
+# Patches dalli to use SHA256 checksum instead of MD5.
+# Once we stop supporting Ruby 2.4, we should upgrade dalli
+# and see if any configuration is needed for same effect.
+#
+# key digests were actually fixed with
+# https://github.com/petergoldstein/dalli/commit/74b2625f11ff56dd6f55c7316bc115bc5a29be5f
+
+Dalli::Client.prepend(Module.new do
```

Comment:
This is some pretty fancy key patching going on here.  It's easy enough to follow, but thoughts on adding a test?  I know it's temporary throw away code, but there's a saying `the most permanent code is temporary code` so might be worth adding a test?

Some examples of how to do this:
https://stackoverflow.com/questions/25147236/how-do-you-test-your-config-initializer-scripts-with-rspec-in-rails

Basically the idea is to create an extra class or similar containing a static method and then test that static method in a test.

Reply:
From what I later read, this is an outdated draft standard so most likely we don't use it. And is removes in upcoming gem versions. Perhaps it was useless to monkey patch it. I don't find it worth spending more time on it.